iot/api-client/manager/manager.py

```diff
@@ -93,6 +94,12 @@ def create_rs256_device(
 ):
     """Create a new device with the given id, using RS256 for
     authentication."""
+    # [START iot_create_rsa_device]
```

Comment:
Please remove the region tags from this file. This file is only being used for test setup. The version displayed to users is in python-iot.

Reply:
ok